cpu/stm32_common/periph/flashpage.c

```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * @ingroup     cpu_stm32f0
+ * @ingroup     cpu_stm32_common
```

Comment:
Actually, (though without having it tested), it should also run fine on the f3 and f4. So I don't like putting this type of comment in -> I would say the first one who configures it for one of these CPUs needs to make sure it will work...